src/lib/services/feature-toggle-service.ts

```diff
@@ -1967,7 +1968,26 @@ class FeatureToggleService {
     }
 
     async updatePotentiallyStaleFeatures(): Promise<void> {
-        await this.featureToggleStore.updatePotentiallyStaleFeatures();
+        const potentiallyStaleFeatures =
+            await this.featureToggleStore.updatePotentiallyStaleFeatures();
+        if (this.flagResolver.isEnabled('emitPotentiallyStaleEvents')) {
+            if (potentiallyStaleFeatures.length) {
+                return this.eventStore.batchStore(
+                    potentiallyStaleFeatures.map(
+                        ({ name, potentiallyStale }) => ({
+                            type: FEATURE_POTENTIALLY_STALE_UPDATED,
+                            createdBy: 'unleash-system',
+                            data: {
+                                name,
+                                potentiallyStale,
+                            },
+                        }),
+                    ),
+                );
+            }
+        } else {
+            console.log('NOT ENABLED\n\n\n\n');
```

Comment:
Can clear this

Reply:
Oh, for sure! Just left that in by mistake.

Reply:
30f1f49e51005a0ddefd7376253c44672a20e1c9